package/src/manifest.rs

```diff
@@ -373,12 +477,18 @@ mod tests {
             // Invalid dependency names
             r#"{name = "foo", version = "1.0.0", minimal_nickel_version = "1.9.0", authors = [], dependencies = { "42" = 'Path "dep" }}"#.as_bytes(),
             r#"{name = "foo", version = "1.0.0", minimal_nickel_version = "1.9.0", authors = [], dependencies = { "has space" = 'Path "dep" }}"#.as_bytes(),
-            // TODO: add index dependencies, once they're supported
+
+            r#"{name = "foo", version = "1.0.0", minimal_nickel_version = "1.9.0", authors = [], dependencies = { dep = 'Index { package = "codeberg/example/example", version = "=1.2.0" }}}"#.as_bytes(),
+            // This should become successful once we support version resolution
+            r#"{name = "foo", version = "1.0.0", minimal_nickel_version = "1.9.0", authors = [], dependencies = { dep = 'Index { package = "github/example/example", version = "1.2.0" }}}"#.as_bytes(),
```

Comment:
Is it ok to keep a failing test? Won't that make the CI red?

Reply:
Ah, but this case is in the "manifests that should fail" test, so the test passes. By "become successful" I meant that it will get moved to the "manifests that should succeed" test.